packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1551,19 +1551,35 @@ idx.error.code.access_denied.device_assurance.remediation.explanation_multiple_r
 idx.error.code.access_denied.device_assurance.remediation.option_index = Option {0}:
 # {0} is the version for this OS, i.e. Update to Android v10
 idx.error.code.access_denied.device_assurance.remediation.android.upgrade_os_version = Update to Android {0}
+idx.error.code.access_denied.device_assurance.remediation.android.upgrade_os_version.at_least = Update to at least Android {0}
```

Comment:
Do we have to have one set of new strings for each OS? Can the OS name (Android, iOS, macOS and Windows) be a variable too? They will not be translated anyway IIUC.

Reply:
Correct, platforms are not translated. But I will keep separate strings for each platform. Otherwise, backend will have to send the platform and list of versions to SIW.

Reply:
We should have the platform and expected osVersions when we need to build the remediation message.